src/independent_mogp.jl

```diff
@@ -115,10 +115,79 @@ Posterior implementation for isotopic inputs, given diagonal Σy (OILMM).
 See AbstractGPs.jl API docs.
 """
 function AbstractGPs.posterior(
-    ft::IsotropicFiniteIndependentMOGP, y::AbstractVector{<:Real}
+    ft::IsotropicByOutputsFiniteIndependentMOGP, y::AbstractVector{<:Real}
 )
     finiteGPs = finite_gps(ft, ft.Σy[1])
     ys = collect(eachcol(reshape(y, (length(ft.x.x), :))))
     ind_posts = [AbstractGPs.posterior(fx, y_i) for (fx, y_i) in zip(finiteGPs, ys)]
     return independent_mogp(ind_posts)
 end
+
+# AbstractGPs APIs implementations for MOInputIsotopicByFeatures.
+
+const IsotropicByFeaturesFiniteIndependentMOGP = FiniteGP{
+    <:IndependentMOGP,<:MOInputIsotopicByFeatures,<:Diagonal{<:Real,<:Fill}
+}
+
+function reorder_features_to_outputs_indices(x::MOInputIsotopicByOutputs)
```

Comment:
This can probably happen after, but should functionality that converts `MOInputIsotopicByOutputs` to `MOInputIsotopicByFeatures` and vice-versa live in `KernelFunctions`?

Reply:
Yeah, that would probably be good. I'll open an issue.

Reply:
👍🏻 , resolving then!